endless-sky/.vscode/tasks.json

```diff
@@ -3,10 +3,7 @@
     "tasks": [
         {
             "label": "ES Data: Run Conversation",
-            "command": "${command:endlesssky.talk}",
-            "dependsOn": [
-                "Build: Compile Endless Sky OpenGL Desktop"
-            ]
+            "command": "${command:endlesssky.talk}"
```

Comment:
endlesssky.talk depends on the endless-sky built binary because it uses `endless-sky --talk` under the hood.

There's also a vscode settings json for the endless-sky path that may need to be updated.  I can address that post-merge